ui/src/router/edit/EditRouterView.js

```diff
@@ -1,19 +1,20 @@
-import React, { useContext, useEffect } from "react";
+import React, { useContext, useEffect, useState } from "react";
 import { TuringRouter } from "../../services/router/TuringRouter";
 import { FormContext, FormContextProvider } from "@gojek/mlp-ui";
 import { addToast, replaceBreadcrumbs, useToggle } from "@gojek/mlp-ui";
 import { UpdateRouterForm } from "../components/form/UpdateRouterForm";
 import { ExperimentEngineContextProvider } from "../../providers/experiments/ExperimentEngineContextProvider";
 import { VersionComparisonView } from "./components/VersionComparisonView";
 import { useTuringApi } from "../../hooks/useTuringApi";
-import { DeploymentSummary } from "../components/form/components/DeploymentSummary";
+import { UpdateSummary } from "../components/form/components/UpdateSummary";
 import { ConfirmationModal } from "../../components/confirmation_modal/ConfirmationModal";
+import { VersionCreationSummary } from "../components/form/components/VersionCreationSummary";
 
 const EditRouterView = ({ projectId, currentRouter, ...props }) => {
-  const { data: updatedRouter } = useContext(FormContext);
+  const { data: routerConfig } = useContext(FormContext);
   const [showDiffView, toggleDiffView] = useToggle();
 
-  const [submissionResponse, submitForm] = useTuringApi(
+  const [updateRouterResponse, updateRouterForm] = useTuringApi(
```

Comment:
Nit: `updateRouterForm` sounds like the form component rather than an actionable function name. Can it be called `updateRouterCallback` or `submitUpdateRouter` ? 

Same for the create version.